runtests.py

```diff
@@ -693,6 +694,22 @@ def build_test(self, test_class, path, workdir, module, tags, language,
                           pythran_dir=pythran_dir)
 
 
+def skip_c(tags):
+    if 'cpp' in tags['tag']:
+        return True
+
+    # We don't want to create a distutils key in the
+    # dictionary so we check before looping.
+    if 'distutils' in tags:
+        for option in tags['distutils']:
+            splitted = option.split('=')
+            if len(splitted) == 2:
+                argument, value = splitted
+                if argument.strip() == 'language' and value.strip() == 'c++':
```

Comment:
Not sure if that makes it simpler, but it might be possible to use Cython's own directives parser here.

Reply:
After a quite long search, I didn't find the Cython parser for the distutils options. I found `Cython.Compiler.Options.parse_directive_list` but it's only the parser for Cython directives (it raises an error for other directives, like `ValueError: Unknown option: "language"`). Can I have at least the file in which there is the equivalent for distutils? Thank you.

Reply:
It's here:
https://github.com/cython/cython/blob/7fd335645550dc800dda60bf4565026dc44c4d1d/Cython/Build/Dependencies.py#L222-L245

Admittedly, it's not very reusable as it stands, and we should avoid importing Cython completely at this point in the test runner (it's only imported later if needed). So, it would need to be moved out, but parts of it are compiled and optimised at build time, which we might lose if we move it e.g. to `Cython.Compiler.Options`, which is the most obvious place, but a file that's better left uncompiled. Tricky, tricky...

Duplicating a part of this in the test runner might not be that bad after all... ;-)

Reply:
Thanks for the insight! I have to admit that I was a bit lost. Indeed, I agree that we should keep those few lines, we'll reconsider it later on if we see that it grows.